src/components/shop-app.js

```diff
@@ -506,22 +471,6 @@ class ShopApp extends Element {
       }
     }
   }
-
```

Comment:
Let's also remove `_domChange` since we don't have that event in lit-element. We can add the perf stuffs if we need later.